main.yml

```diff
@@ -0,0 +1,48 @@
+name: Step-time Biofeedback Project
+
+on:
+  push:
+    branches:  
+      - main
+      - feature/*
+  pull_request:
```

Comment:
@JackMcPhillips1543 there are branch protection rules on main that may hinder your progress with testing this. I'd say just keep it for only feature branches and we can adopt it for main later